aries-site/src/data/structures/templates/templates.js

```diff
@@ -244,6 +245,26 @@ export const templates = [
     sections: [],
     tags: [],
   },
+  {
+    name: 'popover',
+    available: true,
+    cardOrder: 5,
+    description: `A Popover is an overlay that is opened by a
+     click trigger.
+     They are used to provide additional contextual information and might
+     contain interactive elements.`,
+    preview: {
+      component: () => <PopoverPreview />,
+      background: 'background-front',
+    },
+    seoDescription: `A Popover is an overlay that is opened by a
+     click trigger.
+     They are used to provide additional contextual information and might
+     contain interactive elements.`,
+    relatedContent: ['CheckBoxGroup'],
```

Comment:
CheckBoxGroup doesn't seem related to popover

Reply:
oh yeah that probably got moved over when I copied I can take them out